package.json

```diff
@@ -54,10 +53,12 @@
   "dependencies": {
     "broccoli-merge-trees": "^1.1.4",
     "broccoli-replace": "^0.12.0",
-    "ember-cli-babel": "^5.1.9",
-    "ember-cli-htmlbars": "^1.1.0"
+    "ember-cli-babel": "^6.1.0",
+    "ember-cli-htmlbars": "^1.1.1",
+    "broccoli-babel-transpiler": "6.0.0-alpha.1",
```

Comment:
This should be removed (I think) once the other issue is addressed (using ember-cli-babel's `.transpileTree` method).